gno.land/pkg/gnoweb/handler.go

```diff
@@ -341,8 +341,31 @@ func generateBreadcrumbPaths(url *weburl.GnoURL) components.BreadcrumbData {
 		})
 	}
 
-	if args := url.EncodeArgs(); args != "" {
-		data.Args = args
+	// Add args
+	if url.Args != "" {
+		argSplit := strings.Split(url.Args, "/")
+		var nonEmptyArgs []string
+		for _, a := range argSplit {
+			if a != "" {
+				nonEmptyArgs = append(nonEmptyArgs, a)
+			}
+		}
```

Comment:
Nitpick: you could simplify this by using the `DeleteFunc` of the [slices standard package](https://pkg.go.dev/slices#DeleteFunc).